src/app/child-dev-project/previous-schools/previous-schools.component.ts

```diff
@@ -44,6 +56,18 @@ export class PreviousSchoolsComponent
   @Output() changedRecordInEntitySubrecord = new EventEmitter<any>();
   records = new Array<ChildSchoolRelation>();
   columns = new Array<ColumnDescription>();
+  current: ChildSchoolRelation;
+
+  schoolMap: Map<string, School>;
+
+  public config: PreviousRelationsConfig = {
+    single: false,
+    columns: {
+      schoolId: "School",
+      start: "From",
+      end: "End",
+    },
```

Comment:
Shouldn't the default value here match the previous behaviour for the PreviousSchools? If users have a custom config but do not update it, backward compatibility can be preserved then

```suggestion
    single: true,
    columns: {
      schoolId: "School",
      schoolClass: "Class",
      start: "From",
      end: "End",
      "result": "Result",
    },
```